.circleci/unittest/linux_stable/scripts/run_all.sh

```diff
@@ -53,16 +55,11 @@ printf "* Installing dependencies (except PyTorch)\n"
 echo "  - python=${PYTHON_VERSION}" >> "${this_dir}/environment.yml"
 cat "${this_dir}/environment.yml"
 
-
-if [[ $OSTYPE == 'darwin'* ]]; then
-  PRIVATE_MUJOCO_GL=glfw
-elif [ "${CU_VERSION:-}" == cpu ]; then
-  PRIVATE_MUJOCO_GL=osmesa
-else
-  PRIVATE_MUJOCO_GL=egl
-fi
+PRIVATE_MUJOCO_GL=osmesa
```

Comment:
Osmesa is incredibly slow and not sustainable. If the machine has a gpu we need EGL

Reply:
Can we do this in a follow-up? I haven't got this job work with EGL (with errors like gymnasium regkeys not finding the ALE namespace, Cannot initialize a headless EGL display). Makes sense to use EGL if osmesa is not sustainable, but would like to unblock at least some version of this job in order to complete the CCI deprecation.

Re speed, fwiw this job runs in 30 mins and the old job ran for about 39 minutes on CCI.